runtime/onert/core/include/ir/Model.h

```diff
@@ -175,8 +175,26 @@ class Model
 
 private:
   std::shared_ptr<backend::custom::IKernelBuilder> _kernel_builder;
-};
 
+public:
+  void add_metadata(const std::string &name, std::shared_ptr<const ir::Data> data)
```

Comment:
How about following the same naming convention like other functions? Other functions use CamelCase naming convention.

Reply:
Thanks for pointing out.

I tried to, but it's look like that some method uses snake_case, and some uses CamelCase
Is there a more recommended rule? 

https://github.com/zetwhite/ONE/blob/eba3f1d21a0dfdcc600f8d9d8982fae51939bc6c/runtime/onert/core/include/ir/Model.h#L128-L162

Reply:
Well, when I looked onert code again, there is no naming convention rule. So you don't need to update it.